pyproject.toml

```diff
@@ -40,7 +40,6 @@ exclude = [
 testpaths = [
     "tests",
 ]
-addopts = '--cov=omnigibson --cov-report=xml'
```

Comment:
what is this?

Reply:
I am actually not sure... I need to remove this line in order for pytest to work. Unclear why.. @cgokmen Do you have any idea?